### DISCUSSION TITLE
Welcome to ml-survey-service Discussions!

### DISCUSSION THREAD

**Discussion by aks30** [shikshalokam/ml-survey-service#139] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by reenakp2601** [comment]
Hello, @aks30 
I'm a Backend Developer proficient in Node.js, Express.js MongoDB, React.js, MySQL, PostgreSQL, Bootstrap, Database Management, API building & having a little experience with python and Django. I want to work on issues regarding API building and management as I am very much familiar with this. The project you mentioned, "ShikshaLokam," sounds intriguing, and I believe my skills align well with its goals. I'd love to learn more about it and discuss how I can be a part of the community. Looking forward to connecting further!

To understand this project and its code more deeply I want to set up this on my system but, in Readme.md the set-up guide includes- 
Setting the keycloak
Create a folder on service directory named: keycloak-public-keys
Inside that folder create a file GRxxx....................xxxxx60fA
for keycloak file please contact Backend Team
Will you please guide me on this.
